yelp_package/bintray.json.in

```diff
@@ -0,0 +1,33 @@
+{
+    "package": {
+        "name": "paasta-tools",
+        "repo": "paasta",
+        "subject": "yelp",
+        "desc": "An open, distributed platform as a service",
+        "website_url": "paasta.readthedocs.org",
+        "issue_tracker_url": "https://github.com/Yelp/paasta/issues",
+        "vcs_url": "https://github.com/Yelp/paasta.git",
+        "github_use_tag_release_notes": true,
+        "github_release_notes_file": "RELEASE.md",
+        "licenses": ["Apache-2.0"],
+        "labels": [],
+        "public_download_numbers": false,
+        "public_stats": false,
+        "attributes": []
+    },
+
+    "version": {
+        "name": "@PAASTAVERSION@",
+        "desc": "Version @PAASTAVERSION@",
+        "released": "@DATE@",
+        "vcs_tag": "v@PAASTAVERSION@",
+        "attributes": [],
+        "gpgSign": false
```

Comment:
I'm not sure if travis supports this, but we should look into a way to get signed packages at some point.